.gitignore

```diff
@@ -70,6 +70,10 @@ node-client/.virtualenv
 \#*
 .ensime*
 ensime.sbt
+**/**/.bloop
```

Comment:
I know this isn't settled by a coding standard, but if we start adding every imaginable tool to the project ignore file - it's gonna get huge.
maybe instead 
https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
agreed. will modify